lighthouse-core/audits/load-fast-enough-for-pwa.js

```diff
@@ -57,16 +56,21 @@ class LoadFastEnough4Pwa extends Audit {
         // Ignore requests that don't have timing data or resources that have
         // previously been requested and are coming from the cache.
         if (!record._timing || record._fromDiskCache || record._fromMemoryCache) {
-          return undefined;
+          return {url: record._url};
```

Comment:
since we're now using an object we can be more explicit

```js
const ret = {url: record._url};

if (!record._timing)) ret.isMissingTiming = true;
if (record._fromDiskCache || record._fromMemoryCache) ret.isFromCache = true;

if (ret.isFromCache || ret.isMissingTiming) return ret;

ret.latency = record._timing.receiveHeadersEnd - record._timing.sendEnd;
ret.latencyStr = ret.latency.toLocaleString(undefined, {maximumFractionDigits: 2});

return ret;
```